src/test/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicyTest.java

```diff
@@ -151,98 +164,6 @@ public void shouldNotAgeItemToLostIfActualCostIsUsed() {
     assertFalse(lostItemPolicy.canAgeLoanToLost(now(UTC)));
   }
 
-  @Test
```

Comment:
Why have all of these tests been removed? Have they been replaced with similar tests elsewhere or are they no longer applicable?

Reply:
Could some unit tests for the `shouldCloseLoan` method be written to replace these, as parts of that logic don't appear to be tested?

Reply:
They are not needed anymore, the `shouldChargeFeesWhenAgedToLost` method is removed.

Reply:
> Could some unit tests for the shouldCloseLoan method be written to replace these, as parts of that logic don't appear to be tested?

Added.